packages/browser/src/helpers.ts

```diff
@@ -125,18 +136,18 @@ export function wrap(
 
       throw ex;
     }
-  };
-  /* eslint-enable prefer-rest-params */
+  } as unknown as WrappedFunction<T>;
 
+  // Wrap the wrapped function in a proxy, to ensure any other properties of the original function remain available
   // Accessing some objects may throw
   // ref: https://github.com/getsentry/sentry-javascript/issues/1168
   try {
     for (const property in fn) {
       if (Object.prototype.hasOwnProperty.call(fn, property)) {
-        sentryWrapped[property] = fn[property];
+        sentryWrapped[property as keyof T] = fn[property as keyof T];
       }
     }
-  } catch (_oO) {} // eslint-disable-line no-empty
+  } catch {} // eslint-disable-line no-empty
```

Comment:
super-l: Instead of the eslint-disable pragma, we could also just throw a comment into the catch block to explain why we we try/catch and move on (basically the above comment)